src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IParameterReferenceExpression.cs

```diff
@@ -903,7 +885,10 @@ public I F(object x)
                 references: new[] { MscorlibRef, SystemRef, compilation0.EmitToImageReference(embedInteropTypes: true) });
 
             string expectedOperationTree = @"
-IOperation:  (OperationKind.None, IsInvalid) (Syntax: 'new I(x)')
+IInvalidExpression (OperationKind.InvalidExpression, Type: I, IsInvalid) (Syntax: 'new I(x)')
+  Children(2):
+      IParameterReferenceExpression: x (OperationKind.ParameterReferenceExpression, Type: System.Object, IsInvalid) (Syntax: 'x')
+      IOperation:  (OperationKind.None, IsInvalid) (Syntax: 'new I(x)')
```

Comment:
> IOperation:  (OperationKind.None, IsInvalid) (Syntax: 'new I(x)') [](start = 5, length = 66)

I am not sure what this node is about. We should follow up.

Reply:
not 100% sure on what you want to follow.